vllm/engine/arg_utils.py

```diff
@@ -627,6 +634,9 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                                     default='mp',
                                     help='Backend for data parallel, either '
                                     '"mp" or "ray".')
+        parallel_group.add_argument(
+            "--data-parallel-hybrid-lb",
+            **parallel_kwargs["data_parallel_hybrid_lb"])
```

Comment:
Instead of adding `--data-parallel-hybrid-lb`, could we make it so that omitting `--headless` implies `data_parallel_hybrid_lb` 

Is there some issue we run into?

Reply:
I like that idea @tlrmchlsmth, it would fit quite nicely as a generalization of the existing semantics.